src/lib/containers/entity_finder/EntityFinder.tsx

```diff
@@ -0,0 +1,344 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import React, { useEffect, useState } from 'react'
+import { QueryClient, QueryClientProvider } from 'react-query'
+import { useListState } from '../../utils/hooks/useListState'
+import { EntityBundle, EntityHeader } from '../../utils/synapseTypes'
+import { EntityType } from '../../utils/synapseTypes/EntityType'
+
+import { TreeView } from './tree/TreeView'
+import { ReflexContainer, ReflexElement, ReflexSplitter } from 'react-reflex'
+import { Button } from 'react-bootstrap'
+import useGetEntityBundle from '../../utils/hooks/SynapseAPI/useEntityBundle'
+import { SynapseClient } from '../..'
```

Comment:
Here's a puzzle...if any module that rollup bundles imports `SynapseClient` from `lib` and not `lib/utils`, it takes twice as long to run, and results in a bundle that doesn't work (the object ends up not having key properties like SynapseComponents).

1. Why does this happen?
2. How can we prevent it from happening? Because this is an easy mistake to make.